cmd/srcd/cmd/sql.go

```diff
@@ -45,6 +47,24 @@ var sqlCmd = &cobra.Command{
 			query = args[0]
 		}
 
+		c, err := daemon.Client()
+		if err != nil {
+			logrus.Fatalf("could not get daemon client: %v", err)
+		}
+
+		started := logAfterTimeout("this is taking a while, if this is the first time you launch sql client, it might take a few more minutes while we install all the required images")
```

Comment:
I'd define a constant with this text, and use it here; or at least split the line to fit in the line limit.

Reply:
splitted. Defining a constant that is used only once and is warning message doesn't sound good for me at all...

Reply:
good enough. Now it's readable in GH :dagger: thanks